### PR TITLE
Add style

### PR DESCRIPTION
**Pull request by benwmx**
## Details 🔢 
For this milestone we:
* Added the style to the page 
* Added the classes for the Html elements.
demo2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,19 +5,27 @@
...
-  </form>
+  <div class="line"></div>
+  <section class="add-new-book">
+    <h2>Add a new book</h2>
+    <form action="#">
+      <input class="input" type="text" name="title" id="title" placeholder="Title" required>
+      <br>
+      <input class="input" type="text" name="author" id="author" placeholder="Author" required>
+      <br>
+      <input class="submit" type="button" value="Add" id="button">
```
Comment (ZawadiSumba66) [inline index.html:26]: - [ ] The add button is towards the right of the form and not the center. Let's try and follow the wireframe layout as much as possible

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
### Hi Team 😄  

Great job on the styling of your Awesome Books project.
However, there are some issues that you still need to work on to go to the next project but you are almost there! 💪 

 ## REQUIRED CHANGES ♻️ 
- [ ] While you decide to add the screenshot and the live demo to your README file you should ensure that they are showing and/or displaying to maintain professionalism in your repo. Otherwise, you may consider adding later while you learn to fix the issue. This may also avoid misleading other users who will interact with your repo.
- [ ] Check the other comment below.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by benwmx** [comment]
hi, @ZawadiSumba66  thank you for your review, I did all the changes that you requested.

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html:26]: This issue is well resolved ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Rachid & Mahbub, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 🍾 :shipit: .
Congratulations! 🎉 🎉 🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**